near-sdk/compilation_tests/bad_argument.stderr

```diff
@@ -1,3 +1,13 @@
+warning: use of deprecated type alias `json_types::public_key::Base58PublicKey`: PublicKey type is now unified with Base58PublicKey. It is recommended to use PublicKey going forward to avoid using similar sounding types for the same thing.
+  --> $DIR/mod.rs:12:22
+   |
+12 | pub use public_key::{Base58PublicKey, CurveType, PublicKey};
+   |                      ^^^^^^^^^^^^^^^
+   |
+   = note: `#[warn(deprecated)]` on by default
+
+warning: 1 warning emitted
+
```

Comment:
just update the tests to use the new type, the deprecation should not be accounted for in tests

Reply:
AFAIK the test's don't actually use the `Base58PublicKey` type itself. This is just the export and it fails the compilations tests if I don't include it

Reply:
I guess manually exporting it causes this warning to appear in the tests. After moving the deprecated type into `json_types/mod.rs`, it's fine